lib/checkout.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './takeaway.rb'
+
+class Checkout
```

Comment:
This class currently isn't in use. What is the motivation for this class? It currently formats a number into a string, which is something that can be done using the method shown in the link above so perhaps we don't need a separate class for this? What do you think?